readthedocs/projects/forms.py

```diff
@@ -476,7 +476,7 @@ class WebHookForm(forms.Form):
 
     """Project webhook form."""
 
-    url = forms.URLField()
+    url = forms.URLField(max_length=WebHook._meta.get_field('url').max_length)
```

Comment:
Can't we use a ModelForm for this and inherit the validation for url field?

Reply:
Yes, you are right. I will update to a ModelForm.